packages/openapi-generator/src/sdk-metadata/generation-and-usage.ts

```diff
@@ -41,8 +40,7 @@ export async function getGenericGenerationAndUsage(): Promise<GenerationAndUsage
 
 function getGenericUsage(): InstructionWithTextAndHeader {
   return {
-    instructions: genericCodeSample(),
-    text: genericUsageText,
```

Comment:
question: for generic usage, I guess we decided to show static text.
The requirement has been changed, right?